From a96bb7331225d21cd659f020bbb957d101a1a1d6 Mon Sep 17 00:00:00 2001
From: Takeshi Saito <takeshi.saito.xv@renesas.com>
Date: Fri, 18 Jan 2019 08:44:07 +0900
Subject: [PATCH 735/909] mmc: renesas_sdhi: Change HS400 manual calibration
 value for r8a77990

commit b2e044a3602f7f9ffe6c61b6655e6b0b283d85c4 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

According to latest SDHI HW manual, HS400 manual calibration value
for r8a77990 is changed.

offset= 0 (not change)
calibration code= auto calibration value -4

Signed-off-by: Takeshi Saito <takeshi.saito.xv@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mmc/host/renesas_sdhi_core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/host/renesas_sdhi_core.c b/drivers/mmc/host/renesas_sdhi_core.c
index 393e36142939..bd54028ad807 100644
--- a/drivers/mmc/host/renesas_sdhi_core.c
+++ b/drivers/mmc/host/renesas_sdhi_core.c
@@ -278,7 +278,7 @@ static const struct soc_device_attribute sdhi_quirks_match[]  = {
 	{ .soc_id = "r8a77990",
 	  .data = (void *)(HS400_USE_MANUAL_CALIB |
 			   (SH_MOBILE_SDHI_SCC_TMPPORT3_OFFSET_0 << 24) |
-			   (0x2 << 16)), },
+			   (0x4 << 16)), },
 	{/*sentinel*/},
 };
 
-- 
2.17.1

